ibmcloud/SECURE_EXECUTION.md

```diff
@@ -0,0 +1,106 @@
+# Secure Execution Support for Peer Pods on IBM Cloud VPC
+
+IBM Cloud VPC already supports to create Hyper Protect Virtual Server on LinuxONE. And Confidential Computing is enabled by using the [IBM Secure Execution](https://www.ibm.com/docs/en/linux-on-systems?topic=execution-introduction) for Linux technology. For more information, please refer to [Confidential computing with LinuxONE](https://cloud.ibm.com/docs/vpc?topic=vpc-about-se)
+
+To support Secure Execution for Peer Pods on IBM Cloud VPC, we need build a Secure Execution enabled custom VM image for pod VMs, to replace the non-SE custom image. This document describes how to build a SE enabled custom image, based on the exsiting [image build script](./image/build.sh) for IBM Cloud VPC.
+
+## Set up a demo environment without Secure Execution
+
+Follow the [README.md](./README.md) to setup a demo environment on IBM Cloud VPC. We need create a **s390x** architecture based cluster, so please include the following two lines in the `terraform.tfvars` file:
+```
+instance_profile_name = "bz2-2x8"
+image_name = "ibm-ubuntu-18-04-1-minimal-s390x-3"
+```
+
+## Prepare worker node to build SE enabled image
+
+1. **Build genprotimg**
+
+`genprotimg` is used to build an encrypted boot record from a given kernel, initial RAM disk, parameters, and public host-key document.
```

Comment:
`genprotimg` should be already part of recent distributions -- so not necessarily to build it.

Reply:
I have tried to install it via `apt install s390-tools`, but there is no `genprotimg` in the newest version (2.3.0-0ubuntu3.6) on Ubuntu 18.04

Reply:
oh well.. I guess 18.04 is not that recent ;-) Let me check which distros where it is available.